code/modules/urist/_helpers.dm

```diff
@@ -384,3 +384,21 @@
 		if(light_amt < max_light)
 			return 1
 	return 0
+
+//for throwing things from one turf to another
+
+/proc/launch_atom(var/projectile_type, var/turf/start_turf, var/turf/target_turf)
```

Comment:
If you do it like this:
```suggestion
/proc/launch_atom(var/turf/start_turf, var/turf/target_turf, var/projectile_type =<something> )
```
the API might be a bit nicer - if there's a sane default, you only have to specify endpoints.

Reply:
If this is supposed to a generic helper proc, I'm not even sure what a sane default _would_ be. Because this proc doesn't have to be limited to ship weapons; any sort of projectile could use it